pages/article.js

```diff
@@ -44,6 +49,10 @@ const Article = graphql(article)(({ data: { loading, error, Article } }) => {
               <P>
                 von {Article.author}
               </P>
+              <ActionBar
+                url={PUBLIC_BASE_URL + url.pathname + '/' + url.query.slug}
```

Comment:
this looks wrong—probably generates `.../article/slug` (`article` instead of `artikel`)—I think there is a new `asPath` property on next.js `url` object. Try `PUBLIC_BASE_URL + url.asPath`